src/main/java/com/example/CarDealership/CarDealershipApplication.java

```diff
@@ -1,29 +1,55 @@
 package com.example.CarDealership;
 
 import com.example.CarDealership.models.Dealership;
+import com.example.CarDealership.models.Vehicle;
 import com.example.CarDealership.repositories.DealershipRepository;
+import com.example.CarDealership.repositories.VehicleRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 import java.util.List;
+import java.util.Scanner;
 
 @SpringBootApplication
 public class CarDealershipApplication implements CommandLineRunner {
 
 	@Autowired
 	private DealershipRepository dealershipRepository;
+	@Autowired
+	private VehicleRepository vehicleRepository;
 
 	public static void main(String[] args) {
 		SpringApplication.run(CarDealershipApplication.class, args);
 	}
 
 	@Override
 	public void run(String... args) throws Exception {
```

Comment:
Optionally, and I repeat, optionally, you could allow the user to pick which dealership they'd like to see vehicles for. This would require you to set the dealership id, and then alter the queries to make use of the inventory table to grab vehicles for certain dealerships.